src/logging/describeError.ts

```diff
@@ -35,9 +35,9 @@ const describeOrigin = (origin?: MaybeWithMessageAndOrigin) => {
     return ` (Origin: ${describe(origin)})`;
 };
 
-export default (message: string, error: unknown) => {
+export default (error: unknown) => {
     if (error == null) {
-        return message;
+        return String(error);
```

Comment:
Why are we casting `null` to a string here? Could we return something more explicit?

Reply:
In this case the error can be either `null` or `undefined` (notice the double, not triple equals) which are converted to the strings `'null'` or `'undefined'`. 

Usually `null` or `undefined` should not be passed to this function but if it happens I thought it might be helpful to just show which one was passed.